OpenRA.Mods.Common/Traits/Conditions/GrantChargedConditionOnToggle.cs

```diff
@@ -35,38 +35,47 @@ public class GrantConditionOnDeployWithChargeInfo : PausableConditionalTraitInfo
 		[Desc("Cooldown (in ticks) to reach full charge.")]
 		public readonly int ChargeDuration = 500;
 
-		[Desc("The amount of charge that needs to be present for deploy to be issued. " +
+		[Desc("The amount of charge that needs to be present to turn on the condition. " +
 			"If set to -1, threshold is set to full charge. " +
 			"If activated without full charge " + nameof(ConditionDuration) + " is percentally smaller.")]
 		public readonly int ChargeThreshhold = -1;
 
 		[Desc("How long (in ticks) should the condition stay active?")]
 		public readonly int ConditionDuration = 1;
 
-		[Desc("Can " + nameof(DeployedCondition) + " be canceled by followup deploy order?")]
+		[Desc("Can " + nameof(ActivatedCondition) + " be canceled turned off manually?")]
 		public readonly bool CanCancelCondition = false;
 
+		[Desc("Should we interrupt the current activity and wait for it to finish?")]
+		public readonly bool CancelsCurrentOrders = false;
+
 		[CursorReference]
-		[Desc("Cursor to display when able to (un)deploy the actor.")]
-		public readonly string DeployCursor = "deploy";
+		[Desc("Cursor to display when able to trigger a state change.")]
+		public readonly string OrderCursor = "deploy";
```

Comment:
Why "Order"? We don't preface other types of cursors used by other traits.

Reply:
makes sense